db/migrate/20230306100327_add_trgm_index_to_tag_name.rb

```diff
@@ -0,0 +1,24 @@
+# This migration adds a trigram index to the tags table on the name column.
+# This is used to speed up the search for tags by name, this is a heavy
+# sub query when genereating reports where labels are involved.
+#
+# Trigram indexes are used to speed up LIKE queries, they are not used for
+# equality queries. This is because the index is not a btree index, it is a
+# GIN index. This means that the index is not ordered, and so cannot be used
+# for equality queries.
+#
+# Read more: https://www.postgresql.org/docs/current/pgtrgm.html
+
+class AddTrgmIndexToTagName < ActiveRecord::Migration[6.1]
+  # disabling ddl transaction is required for concurrent index creation
+  # https://thoughtbot.com/blog/how-to-create-postgres-indexes-concurrently-in
+  disable_ddl_transaction!
+
+  def change
+    enable_extension 'pg_trgm'
```

Comment:
This should already be enabled from the previous migration. we can skip this line. otherwise looks good to me.